src/pages/People.tsx

```diff
@@ -0,0 +1,67 @@
+import { useParams } from 'react-router-dom';
+import { PeopleItem } from '../components/PeopleItem';
+import { Person } from '../types';
+import { Loader } from '../components/Loader';
+
+export const People: React.FC<{
+  peopleFromServer: Person[] | undefined;
+  errorMessage: string | undefined;
+}> = ({ peopleFromServer, errorMessage }) => {
+  const { slug = '' } = useParams();
+
+  if (peopleFromServer) {
+    return (
+      <div className="container">
+        <h1 className="title">People Page</h1>
+        <div className="block">
+          <div className="box table-container">
+            {errorMessage && (
+              <p
+                data-cy={
+                  peopleFromServer.length === 0
```

Comment:
```suggestion
                  !peopleFromServer.length
```

Reply:
More in chat. This solution was recomend other mentor in chat.